stories/atoms/components/fileInput/FileInput.stories.mdx

```diff
@@ -0,0 +1,479 @@
+              <Typography as="div" fontFamily="brand" fontSize="small" fontWeight="xlight">
+                This is a `large` FileInput that takes 100% of the content
+              </Typography>
+            </div>
+          }
+        />
+      </div>
+    )}
+  </Story>
+</Canvas>
+
+## Errors
+
+Define the behavior of the component when it is in an error state.
+The default value is `false`, set the `error` parameter to `true` to display the component in an error state.
+Provide the `errorMessage` parameter to display an error message into the component.
+
+<Canvas>
+  <Story name="Errors">
+    {() => {
+      const [files, setFiles] = useState([])
+      const [error, setError] = useState(true)
+      const handleDropped = files => {
+        setFiles(files)
+        setError(false)
+      }
+      return (
+        <div>
+          <FileInput
+            label="Drop your files here, or browse"
+            description={
+              <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'center' }}>
+                <Typography as="div" fontFamily="brand" fontSize="small" fontWeight="xlight">
+                  This file input still in error.
+                </Typography>
+              </div>
+            }
+            acceptedFormats={['image/*']}
+            error
+            errorMessage="This is an error message"
+            onDropped={() => {}}
+          />
+          <div
+            style={{
+              margin: '40px'
+            }}
+          />
+          <FileInput
+            label="Drop your files here, or browse"
+            description={
+              <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'center' }}>
+                <Typography as="div" fontFamily="brand" fontSize="small" fontWeight="xlight">
+                  This file input will no longer be in error when the files are dropped.
+                </Typography>
+              </div>
+            }
+            acceptedFormats={['image/*']}
+            error={error}
+            errorMessage="Another error message."
+            onDropped={handleDropped}
+          />
+        </div>
+      )
+    }}
+  </Story>
+</Canvas>
+
+## Callback method `onDropped`
+
+The callback method `onDropped` is called when a file is dropped into the drop zone of the component.
+
+<Canvas>
+  <Story name="Callback method onDropped">
+    {() => {
+      const [message, setMessage] = useState(null)
+      return (
+        <div>
+          <FileInput
+            label="Drop files here"
+            description={
+              <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'center' }}>
+                <Typography as="div" fontFamily="brand" fontSize="small" fontWeight="xlight">
+                  This input display a message on the `onDropped` callback method
+                </Typography>
+              </div>
+            }
+            onDropped={files => {
```

Comment:
Please, do not use lambda methods in `JSX` to fit with our writing convictions 😌